pinot-controller/src/main/java/org/apache/pinot/controller/helix/SegmentStatusChecker.java

```diff
@@ -178,21 +178,17 @@ private void updateSegmentMetrics(String tableNameWithType, Context context) {
         // No online segments in ideal state
         continue;
       }
+      // Skip push in-progress segments
+      SegmentZKMetadata segmentZKMetadata =
+          _pinotHelixResourceManager.getSegmentZKMetadata(tableNameWithType, partitionName);
+      if (segmentZKMetadata != null
+          && segmentZKMetadata.getPushTime() > System.currentTimeMillis() - _waitForPushTimeSeconds * 1000) {
+        continue;
```

Comment:
Add the comment back here saying the push is not yet finished, since any cases reaching this line must be qualified for that.

Reply:
Added the comment